apps/docs/docs/getting-started/cli-framework/prep-work/creating-a-new-project.md

```diff
@@ -4,18 +4,10 @@ sidebar_position: 2
 
 # Creating a new project
 
-:::warning
-
-`create-disploy-app` is not [yet implemented](https://github.com/Disploy/disploy/issues/50).
-
-:::
+## Start by running the command:
```

Comment:
Maybe “Bootstrap a new project” with a description explaining how CDA gets you started with a basic template.